app/controllers/api/vcs.php

```diff
@@ -1137,6 +1137,7 @@
         $repository['pushedAt'] = $repository['pushed_at'] ?? '';
         $repository['organization'] = $installation->getAttribute('organization', '');
         $repository['provider'] = $installation->getAttribute('provider', '');
+        $repository['defaultBranch'] =  $repository["default_branch"] ?? '';
```

Comment:
```suggestion
        $repository['defaultBranch'] =  $repository['default_branch'] ?? '';
```

Same quotes as above for consistency

Reply:
Argh, silly oversight - updated the PR with the requested info.